src/entities/activity/ui/items/ParagraphTextItem.tsx

```diff
@@ -46,7 +48,9 @@ export const ParagraphTextItem = ({
         fontSize="small"
         color="#72777F"
         mr={2}
-      >{`${numCharacters}/${maxResponseLength} characters`}</Box>
+      >
+        {`${numCharacters}/${maxResponseLength} ${t('charactersCount')}`}
```

Comment:
This should take the number of characters as param, please check a string like `time_to_complete_hm` for an example on how to implement. The value is not always guaranteed to appear at the start of the string.